ext/opcache/zend_file_cache.c

```diff
@@ -1371,6 +1371,7 @@ zend_persistent_script *zend_file_cache_script_load(zend_file_handle *file_handl
 	zend_accel_hash_entry *bucket;
 	void *mem, *checkpoint, *buf;
 	int cache_it = 1;
+	unsigned int actaul_checksum;
```

Comment:
This looks like a typo; shouldn't it be `actual_checksum`?

Reply:
Thanks let me fix that.